docs/docs/using-gatsby-image.md

```diff
@@ -0,0 +1,20 @@
+Using gatsby-image to prevent image bloat
+====
+
+I recently introduced a new Gatsby/React component called [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/).
+
+It has some nice tricks that you’d expect from a modern image component. It uses the new [IntersectionObserver API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to cheaply lazy load images. It holds an image’s position so your page doesn’t jump around as images load. It makes it easy to add a placeholder—either a gray background or a blurry version of the image.
```

Comment:
You know what I just realized! I ought to also have included the plugin READme file as a suggestion for what to include in this doc. I think the "Problem" and "Solution" sections of the plugin READme file should follow this paragraph: https://www.gatsbyjs.org/packages/gatsby-image/